package.json

```diff
@@ -1,39 +1,18 @@
 {
-  "name": "elastic-apm-js-base",
+  "name": "elastic-apm-rum",
+  "private": true,
   "version": "3.0.0",
-  "description": "Elastic APM JavaScript agent",
-  "main": "dist/bundles/elastic-apm-js-base.umd.js",
+  "description": "Elastic APM RUM agent",
   "repository": {
     "type": "git",
     "url": "https://github.com/elastic/apm-agent-js-base.git"
   },
   "scripts": {
-    "prepare-patch": "sh ./node_modules/elastic-apm-js-core/dev-utils/prepare-release patch && npm run build && npm run release",
-    "prepare-minor": "sh ./node_modules/elastic-apm-js-core/dev-utils/prepare-release minor && npm run build && npm run release",
-    "prepare-major": "sh ./node_modules/elastic-apm-js-core/dev-utils/prepare-release major && npm run build && npm run release",
-    "release": "npm run push-tags && npm publish && npm run github-release",
-    "push-tags": "git push origin master --follow-tags",
-    "github-release": "node ./scripts/release-github.js",
-    "build": "webpack",
-    "build-dev": "webpack -w",
-    "build-docs": "sh ./scripts/build_docs.sh apm-agent-js-base ./docs ./build",
-    "karma": "karma start",
-    "karma-coverage": "karma start --coverage",
-    "serve": "node ./scripts/run-script.js serveE2e ./ 8000",
-    "changelog": "node node_modules/conventional-changelog-cli/cli.js  -p angular -i CHANGELOG.md -s",
-    "runUnitTests": "node ./scripts/run-script.js runUnitTests",
-    "runNodeTests": "node ./scripts/run-script.js runNodeTests",
-    "buildE2eBundles": "node ./scripts/run-script.js buildE2eBundles",
-    "run-e2e": "node ./scripts/run-script.js runE2eTests",
-    "run-script": "node ./scripts/run-script.js",
-    "test": "npm run lint && npm run build && npm run buildE2eBundles && npm run runNodeTests && npm run runUnitTests && npm run run-e2e && npm run build-docs",
     "lint": "eslint . && node scripts/license-checker",
-    "format": "eslint . --fix"
+    "build-docs": "sh ./scripts/build_docs.sh apm-agent-js-base ./docs ./build",
+    "format": "eslint . --fix",
+    "test": "npm run lint && lerna bootstrap && lerna run test --stream && npm run build-docs"
```

Comment:
bootstrap shouldnt be called during test TBH, before setup one is supposed to run bootstrap and have lerna configure the links.